src/SessionManager/ProgramSettings.h

```diff
@@ -98,6 +101,19 @@ struct ProgramSettings {
         warning_msgs.clear();
         debug_msgs.clear();
     }
+
+    std::string ParseBrowserCommand(const std::string& url) {
```

Comment:
I think `GenerateBrowserCommand` makes more sense here. It's parsing the browser _argument_ to generate a command string

Reply:
Changed.